motion_predict/src/motion_predict.cpp

```diff
@@ -39,6 +39,9 @@ carma_perception_msgs::msg::PredictedState predictState(const geometry_msgs::msg
 
   x(0)=pose.position.x; // Position X
   x(1)=pose.position.y; // Position Y
+
+  // Need a logic here to possible detect whether if twist.linear.x,y
+  // is in map frame. https://github.com/usdot-fhwa-stol/carma-platform/issues/2407
```

Comment:
Can we add a `TODO` here? Will make it easier to search our codebase for TODO's with their included GitHub Issue or Jira Issue links.

Reply:
added